django/core/exceptions.py

```diff
@@ -53,6 +53,22 @@ class DisallowedRedirect(SuspiciousOperation):
     pass
 
 
+class TooManyFieldsSent(SuspiciousOperation):
+    """
+    The number of fields in a POST request exceeded
+    settings.DATA_UPLOAD_MAX_NUMBER_FIELDS.
+    """
+    pass
+
+
+class RequestBodyTooBig(SuspiciousOperation):
```

Comment:
Perhaps we should consider eg `RequestDataTooBig` or similar. After all, its not strictly the *body* size that's been exceeded. (?)